content/source/docs/enterprise-beta/api/workspaces.html.md

```diff
@@ -5,22 +5,23 @@ sidebar_current: "docs-enterprise2-api-workspaces"
 ---
 
 # Workspaces API
-
+Workspaces represent running infrastructure managed by Terraform.
 
 
 ## Create a workspace
 
 This endpoint creates a new workspace in the organization.
 
-**Method**: POST
+| Method | Path           |
+| :----- | :------------- |
+| `POST`  | `/organizations/:organization/workspaces` |
 
-**Path**: /organizations/:organization/workspaces
 
 ### Parameters
 
-- `name` (string: \<required\>) - Specifies the name of the workspace. This must be a alphanumeric and `-`or `_`. This will be used as an identifier and must be unique in the organization.
-- `environment`(string: "default") - Specifies the environment attribute on the workspace.
-- `organization` (string: \<required\>) - Specififes the username or organizaiton name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This is specified in the URL path.
+- `:organization` (`string: <required>`) - Specififes the username or organizaiton name under which to create the workspace. The organization must already exist in the system, and the user must have permissions to create new workspaces. This is specified in the URL path.
```

Comment:
only specifies the organization name (not the user name)

Reply:
or are you just letting this flow through to the user's default org?

Reply:
Yeah, this should be organization name. I was assuming similar behavior as V1 but realized that's not the case.